admin_guide/high_availability.adoc

```diff
@@ -10,50 +10,163 @@
 toc::[]
 
 == Overview
-This document describes how to setup high available services on your
+This topic describes how to set up highly-available services on your
 OpenShift cluster.
 
-== IP Failover
-This section describes how to make any IP based network service highly available using virtual IP addresses and IP failover.
+The Kubernetes link:../architecture/core_objects/kubernetes_model.html#replication-controller[replication controller] ensures that the deployment requirements,
+in particular the number of replicas, are satisfied when the appropriate
+resources are available. When run with two or more replicas, the link:../architecture/core_objects/routing.html[router] can be resilient to failures, providing a highly-available service. Depending on how the router instances are discovered (via a service,
+DNS entry, or IP addresses), this could impose operational requirements to
+handle failure cases when one or more router instances are "unreachable".
```

Comment:
This para and a few other lines in the topic (I won't note each one, but you can check from viewing the "Raw" file in GH) could still be wrapped at 80. Just make sure any related lists or steps aren't accidentally broken in the process.